ISSUE_TEMPLATE.md

```diff
@@ -26,6 +28,10 @@ __Dataset__:
 <!-- description, or if possible, please share an example dataset to reproduce the issue either as Gist with an arangodump, or an arangosh script with db.collection.save({my: "values"}) statements -->
 
 
+__Size of your Dataset on disk__:
```

Comment:
Not sure if this should be separate "headline". We could make this a list like for My Environment perhaps, and dataset size on disk would be one of the points.

Reply:
thought to add it here because the item immediately before is about the description of the dataset, so wanted to somehow keep similar info close together - can be changed, not an issue